src/posts/gold-design-system.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Celebrating the launch of GOLD Design System 🚀
+date: '2021-09-21'
+tags: [announcements, releases]
+description: GOLD Design System has been published and available to be used in place of the Australian Government Design System   
+permalink: posts/{{ title | slug }}/index.html
+---
+
+Today marks the launch of GOLD Design System, replacing the Australian Government Design System.
+
+* 🚀 Docs are available at [https://gold.designsystemau.org/](https://gold.designsystemau.org/)
+* 🚀 Npm packages have been published at [@gold.au](https://www.npmjs.com/org/gold.au)
+* 🌟 The design system has been renamed from Australian Government Design System to GOLD Design System
+
+To continue to benefit from future updates to the Design System, including security patches, see bottom of page for [Migration details](#migration-details). 
+ 
+### Reasons for changing the name to GOLD
+ 
+Some might remember GOLD was the original name of the design system - the Government Open Language of Design.
+
+<img src="/images/previous-gold-docs.png" style="box-shadow: 1px 1px 10px #cccccc; border: 1px solid #e5e5e5;" />
+
+As the curatorship of Australian Government Design System by the Digital Transformation Agency ends, we thought it would be appropriate to bring GOLD back! 
```

Comment:
I like it. Maybe without the explanation point to not put a finer point on it :D haha